src/threedidepth/calculate.py

```diff
@@ -605,6 +592,48 @@ def __call__(self, progress):
         )
 
 
+class ResultAdmin:
+    """
+    args:
+        gridadmin_path (str): Path to gridadmin.h5 file.
+        results_3di_path (str): Path to (aggregate_)results_3di.nc file.
+
+    Wraps either GridH5ResultAdmin or GridH5AggregateResultAdmin, based on the
+    result type of the file at results_3di_path . Also has custom properties
+    `result_type`, `variable` and `calculation_steps`.
+    """
+
+    def __init__(self, gridadmin_path, results_3di_path):
+        with h5py.File(results_3di_path) as h5:
```

Comment:
Uit mijn hoofd:
calculation_steps = len(ra.nodes.timeseries(indexes=slice(None)).timestamps)

Reply:
Niet helemaal, want bij de aggregate results is die timestamps een dict. Op zich wel netter om het zo te doen als je voorstelt, alleen vind ik het jammer datt ik dan de array moet lezen. Gaan jullie de documentatie voor threedigrid ook nog een beetje uitbreiden? Er is nu niks te vinden over de AggregateResultAdmin.

Reply:
Ja dat is wel het plan, die docs kunnen nog wel een stuk beter.